src/main/java/com/transportcompany/transportcompany/controllers/OrderController.java

```diff
@@ -0,0 +1,43 @@
+package com.transportcompany.transportcompany.controllers;
+
+import com.transportcompany.transportcompany.models.dtos.OrderDTO;
+import com.transportcompany.transportcompany.models.entities.Order;
+import com.transportcompany.transportcompany.services.OrderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/orders")
+public class OrderController {
+
+    @Autowired
+    private OrderService orderService;
+
+    public OrderController(OrderService orderService) {
+        this.orderService = orderService;
+    }
+
+    @PostMapping
+    public ResponseEntity<OrderDTO> createOrder(@RequestBody OrderDTO orderDTO, Object OrderDTO) {
+        return ResponseEntity.ok(orderService.createOrder(orderDTO));
+    }
+
+    @GetMapping("api/order")
+    public ResponseEntity<List<OrderDTO>> getOrders() {
+        List<OrderDTO> orders = orderService.getOrders();
+        return ResponseEntity.ok(orders);
```

Comment:
What if we use `orderService.getOrders()` inside `ok()` method?
Please check your code with SonarLint before creating a PR. Sonar will detect these code smells and instruct you on how to improve the code design.